app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -1114,15 +1116,45 @@ private void updateActionBarSpinner() {
         @NonNull
         @Override
         public View getView(int position, View convertView, @NonNull ViewGroup parent) {
-          convertView = super.getView(position, convertView, parent);
-          SpinnerHolder holder = (SpinnerHolder) convertView.getTag();
-          int page = getCurrentPage();
-
-          String sura = QuranInfo.getSuraNameFromPage(PagerActivity.this, page, true);
-          holder.title.setText(sura);
-          String desc = QuranInfo.getPageSubtitle(PagerActivity.this, page);
-          holder.subtitle.setText(desc);
-          holder.subtitle.setVisibility(View.VISIBLE);
+          int type = super.getItemViewType(position);
+          if (type == 1) {
+            LayoutInflater inflater = getLayoutInflater();
+            convertView = inflater.inflate(R.layout.translation_ab_spinner_selected, parent, false);
+          }else{
+            convertView = super.getView(position, convertView, parent);
+            SpinnerHolder holder = (SpinnerHolder) convertView.getTag();
```

Comment:
thanks, could you clarify this, I didn't need to change anything here after implementing your other comments. Jzk